scripts/package/linux/ubuntu/xenial/build.sh

```diff
@@ -15,7 +15,7 @@ git clone https://github.com/lcm-proj/lcm
 mkdir lcm-build
 pushd lcm-build
 # Forces the package to be installed in /usr/local
-# to be consistent with libbot and the fact these packages
+# to be consistent with libbot2 and the fact these packages
 # are not "official".
 cmake -DBUILD_SHARED_LIBS:BOOL=ON \
       -DCPACK_PACKAGING_INSTALL_PREFIX:PATH=/usr/local \
```

Comment:
Can this install to `/opt/lcm/<version>`?

Reply:
Removing this here as the default location should be correct (and it is easier to handle the version in the `CMakeLists.txt` file).